draft-ietf-taps-impl.md

```diff
@@ -556,10 +556,19 @@ not using TCP), the action should fail gracefully. The application may be inform
 The Transport Services implementation should allow protocol instances in the Protocol Stack to pass up arbitrary generic or protocol-specific
 errors that can be delivered to the application as Soft Errors. These allow the application to be informed of ICMP errors, and other similar events.
 
+## Pooled Connection {#pooled-connections}
+
+For protocols that employ request/response pairs and do not require in-order delivery of the responses, like HTTP/2, interactions can be automatically distributed across several underlying transport connections. 
```

Comment:
I would prefer "For application protocols that employ" instead of just "For protocols that employ", to be even less ambiguous.

Reply:
I somewhat agree, but as HTTP has become a transport for more or less everything, I prefer being unspecific here.

Reply:
I think you should remove "automatically". It is the implementation that distributes I guess so it is not automatic?

Reply:
I mean "automatically from perspective of the application" so yes, the automatism has to be realised by the implementation.
If that is unclear, I can add "by the transport system implementation"

Reply:
As this is the implementation draft, I think the perspective should be from the implementation, so perhaps "the transport implementation may choose to distribute interactions across several underlying transport connections"?

Reply:
works for me.